### PR TITLE
website/docs: add access requests doc

### PR DESCRIPTION
**Pull request by dewi-tik**
### What does this PR change?

Adds docs for new access requests feature

---

## Checklist

-   [x] The project has been linted, built, and tested (`make all`)
-   [x] The documentation has been updated and formatted (`make docs`)

### CODE REVIEW

File: website/docs/add-secure-apps/access-requests/index.mdx [hunk 1]
```diff
@@ -0,0 +1,161 @@
+---
+title: Access requests
+description: "Let users request time-bound access to applications and entitlements, with request rules, expiry, and a full audit trail"
+sidebar_label: Access requests
+authentik_enterprise: true
+authentik_version: "2026.8"
+authentik_preview: true
```
Comment (BeryJu) [inline website/docs/add-secure-apps/access-requests/index.mdx:7]: @BeryJu add preview banner in UI

File: website/docs/add-secure-apps/access-requests/index.mdx [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+- Rule-based routing to the appropriate reviewers.
+- Time-limited access grants that expire automatically, without manual cleanup.
+- Audit events for request creation, approval, denial, and expiry.
+
+## Terminology
+
+| Term                 | Description                                                                                                                  |
+| -------------------- | ---------------------------------------------------------------------------------------------------------------------------- |
+| Access request       | A request from a specific user for access to an application. A request can be pending, approved, denied, or expired.         |
+| Request rule         | A configuration that determines whether a request flow is used, who reviews requests, and how reviewers are notified.        |
```
Comment (BeryJu) [inline website/docs/add-secure-apps/access-requests/index.mdx]: no whether but rather which request flow is used, defaults back to brand default one

Comment (BeryJu) [inline website/docs/add-secure-apps/access-requests/index.mdx]: Also configures who can approve/deny

File: website/docs/add-secure-apps/access-requests/index.mdx [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+| Request rule binding | A binding between a request rule and a requestable application. It also determines which users or groups can request access. |
+| Reviewer             | A user or group authorized to approve or deny an access request.                                                             |
+| Expiry               | The period for which access granted through an approved request remains valid.                                               |
+
+## Access request lifecycle
+
+1. A user logs in to authentik, opens the **Discover** page, and selects a requestable application.
```
Comment (BeryJu) [inline website/docs/add-secure-apps/access-requests/index.mdx]: And optionally selects an application entitlement

File: website/docs/add-secure-apps/access-requests/index.mdx [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+2. authentik runs the configured request flow, if any, to collect required information such as a justification.
+3. authentik creates the access request and evaluates it against the request rules bound to the selected application.
```
Comment (BeryJu) [inline website/docs/add-secure-apps/access-requests/index.mdx]: Actually happens before the flow

File: website/docs/add-secure-apps/access-requests/index.mdx [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+
+## Request rules
+
+Request rules determine whether a request flow is used, who reviews requests, and how reviewers are notified. Users, groups, and policies are bound to a request rule to determine who is authorized to review access requests.
+
+Request rules are bound to an application, which makes the application requestable via the **Discover** page in the [user interface](../../users-sources/user/user-interface.mdx). Users, groups, and policies can be bound to request rule bindings to determine which users are eligible to request access to the application.
+
+## Request flows
+
+Submitting a request can optionally execute a request flow. Use this to collect a justification, require re-authentication or MFA before elevation, or run policies against the requester.
```
Comment (BeryJu) [inline website/docs/add-secure-apps/access-requests/index.mdx]: always executes a flow

File: website/docs/add-secure-apps/access-requests/index.mdx [hunk 6]
```diff
@@ -0,0 +1,161 @@
...
+
+Configure a **Request flow** on either:
+
+- The request rule itself.
+- As a default for the [brand](../../customize/branding/index.md). Used as a fallback when the request rule does not have specify a request flow.
+
+authentik includes a `default-request` flow that is empty. This flow can be configured with any required stages and policies.
```
Comment (BeryJu) [inline website/docs/add-secure-apps/access-requests/index.mdx]: todo @BeryJu add a default request-flow that asks for justification with a prompt field

File: website/docs/add-secure-apps/access-requests/index.mdx [hunk 7]
```diff
@@ -0,0 +1,161 @@
...
+
+**[VERIFY/TODO]** more info on ideas for request flow
```
Comment (BeryJu) [inline website/docs/add-secure-apps/access-requests/index.mdx]:
```suggestion
```

File: website/docs/add-secure-apps/access-requests/index.mdx [hunk 8]
```diff
@@ -0,0 +1,161 @@
...
+
+## Reviewer experience
+
+### Approving and denying access requests
+
+Reviewers see pending requests on the **Discover** page of the [user interface](../../users-sources/user/user-interface.mdx) under the **For My Review** tab. The requester, the requested application, and the time since the request was made are listed. Reviewers click the **Review** button and can approve or deny the request, and optionally add **Notes** before clicking the **Submit** button.
+
+**[VERIFY/TODO]** more info on **Notes**
+
+### Admin interfac
```
Comment (BeryJu) [inline website/docs/add-secure-apps/access-requests/index.mdx]:
```suggestion
### Admin interface
```

File: website/docs/add-secure-apps/access-requests/index.mdx [hunk 9]
```diff
@@ -0,0 +1,161 @@
...
+
+See [Events](../../sys-mgmt/events/index.md) and [Notifications](../../sys-mgmt/events/notifications.md) for more information.
+
+## Considerations and limitations
+
+- A user cannot approve their own request.
+- Approved access is time-bound; plan expiry durations so that they match how long the work actually takes, or users will re-request frequently.
+- Request rules depend on policies. A misconfigured policy can leave requests with no eligible reviewer.
+- **[VERIFY/TODO]** behavior when a request rule or application is deleted while requests are pending.
+- **[VERIFY/TODO]** behavior when an approved user is deactivated before expiry.
```
Comment (BeryJu) [inline website/docs/add-secure-apps/access-requests/index.mdx]:
```suggestion
```

File: website/docs/add-secure-apps/access-requests/index.mdx [hunk 10]
```diff
@@ -0,0 +1,161 @@
...
+Every step of the request lifecycle is recorded as an event, and the new event actions are selectable in event matcher policies. Combine an event matcher policy with a notification rule to alert reviewers via Slack, email, or a webhook when a request is created, and to alert security teams when access is granted.
+
+See [Events](../../sys-mgmt/events/index.md) and [Notifications](../../sys-mgmt/events/notifications.md) for more information.
+
+## Considerations and limitations
+
+- A user cannot approve their own request.
+- Approved access is time-bound; plan expiry durations so that they match how long the work actually takes, or users will re-request frequently.
+- Request rules depend on policies. A misconfigured policy can leave requests with no eligible reviewer.
+- **[VERIFY/TODO]** behavior when a request rule or application is deleted while requests are pending.
```
Comment (BeryJu) [inline website/docs/add-secure-apps/access-requests/index.mdx]: @BeryJu confirm this, you'd probably end up with either an empty request or a request that can never be approved

File: website/docs/add-secure-apps/access-requests/index.mdx [hunk 11]
```diff
@@ -0,0 +1,160 @@
...
```
Comment (dominic-r) [inline website/docs/add-secure-apps/access-requests/index.mdx]: ;)

File: website/docs/add-secure-apps/access-requests/index.mdx [hunk 12]
```diff
@@ -0,0 +1,160 @@
...
+Each request is evaluated against configured request rules and routed to one or more reviewers. When approved, access is granted for a defined period and automatically revoked when it expires.
+
+This supports just-in-time and least-privilege access models. Users retain minimal standing access and request elevated access only when necessary, while authentik records every request and decision for auditing.
+
+Access requests provide:
+
+- A self-service way for users to request access through the [user interface](../../users-sources/user/user-interface.mdx).
+- Rule-based routing to the appropriate reviewers.
+- Time-limited access grants that expire automatically, without manual cleanup.
+- Audit events for request creation, approval, denial, and expiry.
```
Comment (dominic-r) [inline website/docs/add-secure-apps/access-requests/index.mdx]: where's expiry? I only seee https://github.com/goauthentik/authentik/blob/main/authentik/events/models.py#L139-L142

File: website/docs/add-secure-apps/access-requests/index.mdx [hunk 13]
```diff
@@ -0,0 +1,160 @@
...
+- **[VERIFY/TODO]** behavior when a request rule or application is deleted while requests are pending.
+
+## Troubleshooting
+
+| Symptom                                       | Cause and resolution                                                                                                |
+| --------------------------------------------- | ------------------------------------------------------------------------------------------------------------------- |
+| An application does not appear as requestable | The user already has access, or the user is not bound to the request rule binding on the application                |
+| A request has no reviewers                    | No rule is bound to the application, the bound policies did not pass, or the only matched reviewer is the requester |
+| A reviewer cannot action a request            | The reviewer is the requester, or is not matched by the request rule's bindings                                     |
+| Access disappears unexpectedly                | The expiry elapsed. Check the events log for the expiry event and adjust the configured duration                    |
```
Comment (dominic-r) [inline website/docs/add-secure-apps/access-requests/index.mdx]: same as 23

File: website/docs/add-secure-apps/access-requests/index.mdx [hunk 14]
```diff
@@ -0,0 +1,160 @@
...
+- A self-service way for users to request access through the [user interface](../../users-sources/user/user-interface.mdx).
+- Rule-based routing to the appropriate reviewers.
+- Time-limited access grants that expire automatically, without manual cleanup.
+- Audit events for request creation, approval, denial, and expiry.
+
+## Terminology
+
+| Term                 | Description                                                                                                                                                             |
+| -------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Access request       | A request from a specific user for access to an application. A request can be pending, approved, denied, or expired.                                                    |
```
Comment (dominic-r) [inline website/docs/add-secure-apps/access-requests/index.mdx]: same as 23

File: website/docs/add-secure-apps/access-requests/index.mdx [hunk 15]
```diff
@@ -0,0 +1,160 @@
...
+| Request rule         | A configuration that determines which request flow is used, who reviews requests, and how reviewers are notified. If no request flow is set, falls back to brand value. |
+| Request rule binding | A binding between a request rule and a requestable application. It also determines which users or groups can request access.                                            |
+| Reviewer             | A user or group authorized to approve or deny an access request.                                                                                                        |
+| Expiry               | The period for which access granted through an approved request remains valid.                                                                                          |
+
+## Access request lifecycle
+
+1. A user logs in to authentik, opens the **Discover** page, and selects a requestable application, and optionally selects an application entitlement.
+2. authentik creates the access request and evaluates it against the request rules bound to the selected application.
```
Comment (dominic-r) [inline website/docs/add-secure-apps/access-requests/index.mdx:38]: steps 2-3 are inversed. from my and claude's understanding:

the flow runs first, the request is created by [GrantRequestFinalStageView](https://github.com/goauthentik/authentik/blob/3cb4b9953ef31a267104000df98166594ea0d07f/authentik/enterprise/requests/stage.py#L17) at the end of the flow (https://github.com/goauthentik/authentik/blob/3cb4b9953ef31a267104000df98166594ea0d07f/authentik/enterprise/requests/stage.py#L19), which is also why prompt data lands in requester_data. As this is written, the third step collects a justification after step 2 already created and routed the request, which is backwards.

File: website/docs/add-secure-apps/access-requests/index.mdx [hunk 16]
```diff
@@ -0,0 +1,160 @@
+---
+title: Access requests
```
Comment (dominic-r) [inline website/docs/add-secure-apps/access-requests/index.mdx:2]: i dont see any docs about revocation? 

* admin clicking revoke
* Requesters canceling their own pending action

File: website/docs/add-secure-apps/access-requests/index.mdx [hunk 17]
```diff
@@ -0,0 +1,160 @@
...
+| Access request       | A request from a specific user for access to an application. A request can be pending, approved, denied, or expired.                                                    |
+| Request rule         | A configuration that determines which request flow is used, who reviews requests, and how reviewers are notified. If no request flow is set, falls back to brand value. |
+| Request rule binding | A binding between a request rule and a requestable application. It also determines which users or groups can request access.                                            |
+| Reviewer             | A user or group authorized to approve or deny an access request.                                                                                                        |
+| Expiry               | The period for which access granted through an approved request remains valid.                                                                                          |
+
+## Access request lifecycle
+
+1. A user logs in to authentik, opens the **Discover** page, and selects a requestable application, and optionally selects an application entitlement.
```
Comment (dominic-r) [inline website/docs/add-secure-apps/access-requests/index.mdx]: 2 ands chained, drop the second comma+and

File: website/docs/add-secure-apps/access-requests/index.mdx [hunk 18]
```diff
@@ -0,0 +1,160 @@
...
+Request rules are bound to an application, which makes the application requestable via the **Discover** page in the [user interface](../../users-sources/user/user-interface.mdx). Users, groups, and policies can be bound to request rule bindings to determine which users are eligible to request access to the application.
+
+## Request flows
+
+Submitting a request executes a request flow. Use this to collect a justification, require re-authentication or MFA before elevation, or run policies against the requester.
+
+Configure a **Request flow** on either:
+
+- The request rule itself.
+- As a default for the [brand](../../customize/branding/index.md). Used as a fallback when the request rule does not have specify a request flow.
```
Comment (dominic-r) [inline website/docs/add-secure-apps/access-requests/index.mdx]:
```suggestion
- As a default for the [brand](../../customize/branding/index.md). Used as a fallback when the request rule does not specify a request flow.
```

File: website/docs/add-secure-apps/access-requests/index.mdx [hunk 19]
```diff
@@ -0,0 +1,160 @@
...
+
+### Approving and denying access requests
+
+Reviewers see pending requests on the **Discover** page of the [user interface](../../users-sources/user/user-interface.mdx) under the **For My Review** tab. The requester, the requested application, and the time since the request was made are listed.
+
+Reviewers click the **Review** button and can approve or deny the request, and optionally add **Notes** before clicking the **Submit** button. Reviewers also see any prompt data that a requester entered in the request flow, in the **Requester notes** field.
+
+### Admin interface
+
+Administrators can view all access requests via the access requests page (**Applications** > **Access Requests**). Each row represents an access request and shows the request's: requester, creation date and time, requested application, and status. Each access request can be expanded by clicking its chevron (**>**) to show **Requester data**. This is all the prompt data that the requester entered during the request flow, for example a justification for why the require access.
```
Comment (dominic-r) [inline website/docs/add-secure-apps/access-requests/index.mdx]:
```suggestion
Administrators can view all access requests via the access requests page (**Applications** > **Access Requests**). Each row represents an access request and shows the requester, creation date and time, requested application, and status. Each access request can be expanded by clicking its chevron (**>**) to show **Requester data**. This is all the prompt data that the requester entered during the request flow, for example a justification for why they require access.
```

File: website/docs/add-secure-apps/access-requests/index.mdx [hunk 20]
```diff
@@ -0,0 +1,160 @@
...
+
+If the administrator account is set as a reviewer in the associated request rule, they can click **Fulfill** and approve or deny the access request.
+
+## Events and auditing
+
+Every step of the request lifecycle is recorded as an event, and the new event actions are selectable in event matcher policies. Combine an event matcher policy with a notification rule to alert reviewers via Slack, email, or a webhook when a request is created, and to alert security teams when access is granted.
```
Comment (dominic-r) [inline website/docs/add-secure-apps/access-requests/index.mdx]: very long, please split

File: website/docs/add-secure-apps/access-requests/index.mdx [hunk 21]
```diff
@@ -0,0 +1,160 @@
...
+
+See [Events](../../sys-mgmt/events/index.md) and [Notifications](../../sys-mgmt/events/notifications.md) for more information.
+
+## Considerations and limitations
+
+- A user cannot approve their own request.
+- Approved access is time-bound; plan expiry durations so that they match how long the work actually takes, or users will re-request frequently.
```
Comment (dominic-r) [inline website/docs/add-secure-apps/access-requests/index.mdx]: the "or users will" consequence reads as a warning on a guidance,mabye split in 2 sentences?

File: website/docs/add-secure-apps/access-requests/index.mdx [hunk 22]
```diff
@@ -0,0 +1,160 @@
+---
```
Comment (dominic-r) [inline website/docs/add-secure-apps/access-requests/index.mdx:1]: pretty sure this is the only one without a sidebar_position

Comment (dominic-r) [inline website/docs/add-secure-apps/access-requests/index.mdx:1]: no jsx, can we rename to .md? (or just as a wild idea we say fuggedaboutit and rename all .md to mdx)

Comment (dominic-r) [inline website/docs/add-secure-apps/access-requests/index.mdx:1]: all mentions of user inerface linking to user interface doc: we need to mention the new stuff in this release

File: website/docs/add-secure-apps/access-requests/index.mdx [hunk 23]
```diff
@@ -0,0 +1,160 @@
...
+- Rule-based routing to the appropriate reviewers.
+- Time-limited access grants that expire automatically, without manual cleanup.
+- Audit events for request creation, approval, denial, and expiry.
+
+## Terminology
+
+| Term                 | Description                                                                                                                                                             |
+| -------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Access request       | A request from a specific user for access to an application. A request can be pending, approved, denied, or expired.                                                    |
+| Request rule         | A configuration that determines which request flow is used, who reviews requests, and how reviewers are notified. If no request flow is set, falls back to brand value. |
```
Comment (dominic-r) [inline website/docs/add-secure-apps/access-requests/index.mdx]: > If no request flow is set, falls back to brand value.

brand value is vague and sentence feels weird

File: website/docs/add-secure-apps/access-requests/index.mdx [hunk 24]
```diff
@@ -0,0 +1,160 @@
...
+- A user cannot approve their own request.
+- Approved access is time-bound; plan expiry durations so that they match how long the work actually takes, or users will re-request frequently.
+- Request rules depend on policies. A misconfigured policy can leave requests with no eligible reviewer.
+- **[VERIFY/TODO]** behavior when a request rule or application is deleted while requests are pending.
+
+## Troubleshooting
+
+| Symptom                                       | Cause and resolution                                                                                                |
+| --------------------------------------------- | ------------------------------------------------------------------------------------------------------------------- |
+| An application does not appear as requestable | The user already has access, or the user is not bound to the request rule binding on the application                |
```
Comment (dominic-r) [inline website/docs/add-secure-apps/access-requests/index.mdx]: needs periods per style guide and other table

Comment (dominic-r) [inline website/docs/add-secure-apps/access-requests/index.mdx]: from what I saw, it never excludes objects the user can already reach. worth asking jens

File: website/docs/add-secure-apps/access-requests/index.mdx [hunk 25]
```diff
@@ -0,0 +1,160 @@
...
+
+## Configure access requests
+
+### Create a request rule
+
+1. Log in to authentik as an administrator and open the Admin interface.
+2. Navigate to **Applications** > **Request rules**.
+3. Click **New Request Rule** and set the following fields:
+    - **Rule name**: The name for the rule.
+    - **Minimum reviewers**: The minimum number of reviewers who must approve a request.
```
Comment (dominic-r) [inline website/docs/add-secure-apps/access-requests/index.mdx:70]: per https://github.com/goauthentik/authentik/blob/3cb4b9953ef31a267104000df98166594ea0d07f/authentik/enterprise/requests/models.py#L245 ; a single approval from any individually named reviewer should satisfy the rule, regardless of min reviewers. the threshold only applies to reviewers matched via group bindings. 

and well, if theres no group bindings, only a direct review can satisfy it.

File: website/docs/add-secure-apps/access-requests/index.mdx [hunk 26]
```diff
@@ -0,0 +1,160 @@
...
+4. The assigned reviewers are notified based on the request rule's configuration, and the request appears in their pending-request queue.
+5. A reviewer approves or denies the request. Requesters cannot approve their own requests.
+6. If approved, authentik grants access to the application and starts the expiry timer.
+7. When the access period ends, authentik automatically revokes access.
+
+## Request rules
+
+Request rules determine whether a request flow is used, who reviews requests, and how reviewers are notified. Users, groups, and policies are bound to a request rule to determine who is authorized to review access requests.
+
+Request rules are bound to an application, which makes the application requestable via the **Discover** page in the [user interface](../../users-sources/user/user-interface.mdx). Users, groups, and policies can be bound to request rule bindings to determine which users are eligible to request access to the application.
```
Comment (dominic-r) [inline website/docs/add-secure-apps/access-requests/index.mdx]:
```suggestion
Request rules are bound to an application, which makes the application requestable via the **Discover** page in the [user interface](../../users-sources/user/user-interface.mdx). Users, groups, and policies can be bound to request rule bindings to determine which users are eligible to request access to the application. If a request rule binding has no users, groups, or policies bound to it, every user is eligible to request access to the application, following the same convention as application access. Reviewer eligibility works differently: a request rule with no users, groups, or policies bound to it has no one who can approve requests against it.

```

Comment (dominic-r) [reply website/docs/add-secure-apps/access-requests/index.mdx]: https://github.com/goauthentik/authentik/blob/3cb4b9953ef31a267104000df98166594ea0d07f/authentik/enterprise/requests/api/apps.py#L44-L46
https://github.com/goauthentik/authentik/blob/3cb4b9953ef31a267104000df98166594ea0d07f/authentik/enterprise/requests/models.py#L141-L144

File: website/docs/add-secure-apps/access-requests/index.mdx (hunk 17 above)
Comment (dominic-r) [inline website/docs/add-secure-apps/access-requests/index.mdx]: not exactly.

0: requests the app https://github.com/goauthentik/authentik/blob/3cb4b9953ef31a267104000df98166594ea0d07f/web/src/user/requests/BrowseRequestable.ts#L95
exactly 1: auto-requests that entitlement with no prompt https://github.com/goauthentik/authentik/blob/3cb4b9953ef31a267104000df98166594ea0d07f/web/src/user/requests/BrowseRequestable.ts#L83
2+: opens the **Select entitlement** modal. https://github.com/goauthentik/authentik/blob/3cb4b9953ef31a267104000df98166594ea0d07f/web/src/user/requests/BrowseRequestable.ts#L89

File: website/docs/add-secure-apps/access-requests/index.mdx [hunk 27]
```diff
@@ -0,0 +1,160 @@
...
+authentik includes a `default-request` flow that is empty. This flow can be configured with any required stages and policies. Any prompt data entered by a requester during a request flow is shown to the reviewer in the **Requester notes** field. Request flows can be left empty if you don't require user interaction.
+
+## Configure access requests
+
+### Create a request rule
+
+1. Log in to authentik as an administrator and open the Admin interface.
+2. Navigate to **Applications** > **Request rules**.
+3. Click **New Request Rule** and set the following fields:
+    - **Rule name**: The name for the rule.
```
Comment (dominic-r) [inline website/docs/add-secure-apps/access-requests/index.mdx]:
```suggestion
    - **Rule Name**: The name for the rule.
```

https://github.com/goauthentik/authentik/blob/3cb4b9953ef31a267104000df98166594ea0d07f/web/src/admin/requests/RequestRuleForm.ts#L71

File: website/docs/add-secure-apps/access-requests/index.mdx (hunk 19 above)
Comment (dominic-r) [inline website/docs/add-secure-apps/access-requests/index.mdx]:
```suggestion
Administrators can view all access requests via the access requests page (**Applications** > **Access Requests**). Each row represents an access request and shows the request's: requester, creation date and time, requested application, and status. Each access request can be expanded by clicking its chevron (**>**) to show **Requester Data** and **Targets**. This is all the prompt data that the requester entered during the request flow, for example a justification for why the require access.
```

https://github.com/goauthentik/authentik/blob/3cb4b9953ef31a267104000df98166594ea0d07f/web/src/admin/requests/AccessRequestListPage.ts#L64-L72

File: website/docs/add-secure-apps/access-requests/index.mdx (hunk 22 above)
Comment (dewi-tik) [reply website/docs/add-secure-apps/access-requests/index.mdx:1]: I'm happy with changing them all to mdx, makes any future changes a bit easier

File: website/docs/add-secure-apps/access-requests/index.mdx (hunk 19 above)
Comment (dominic-r) [reply website/docs/add-secure-apps/access-requests/index.mdx]: Also, to add to this comment, would be better if u used fields in the ui instead https://github.com/goauthentik/authentik/blob/3cb4b9953ef31a267104000df98166594ea0d07f/web/src/admin/requests/AccessRequestListPage.ts#L56-L62

File: website/docs/add-secure-apps/access-requests/index.mdx [hunk 28]
```diff
@@ -0,0 +1,160 @@
...
+3. Expand the request rule by clicking the chevron (**>**) next to its name.
+4. Click **Create or bind...**.
+5. Select the users, groups, or policies that will review access requests for that request rule.
+   You can bind as many users, groups, and policies as necessary.
+
+### Make an application requestable
+
+1. Log in to authentik as an administrator and open the Admin interface.
+2. Navigate to **Applications** > **Applications** and click the application name that you want to make requestable.
+3. Open the **Request rules** tab, click **Bind existing rule** and set the following fields:
```
Comment (dominic-r) [inline website/docs/add-secure-apps/access-requests/index.mdx]:
```suggestion
3. Open the **Request rules** tab, click **Create or bind...**, then select **Bind existing rule**
```

File: website/docs/add-secure-apps/access-requests/index.mdx [hunk 29]
```diff
@@ -0,0 +1,160 @@
...
+2. Navigate to **Applications** > **Applications** and click the application name that you want to make requestable.
+3. Open the **Request rules** tab, and expand a request rule by clicking the chevron (**>**) next to its name.
+4. Click **Create or bind...**.
+5. Select the users, groups, or policies that can request access to the application via that request rule.
+   You can bind as many users, groups, and policies as necessary.
+
+### Verify the configuration
+
+1. Log in as a test user who does not have access to the application.
+2. Open the **Discover** tab and confirm that the application is offered as requestable.
```
Comment (dominic-r) [inline website/docs/add-secure-apps/access-requests/index.mdx]:
```suggestion
2. Open the **Discover** page and confirm that the application is offered as requestable.
```

Discover is the nav item

the tabs are browse, my requests, for review. Inconsistent with L37 120 124 130.

File: website/docs/add-secure-apps/access-requests/index.mdx (hunk 20 above)
Comment (dominic-r) [inline website/docs/add-secure-apps/access-requests/index.mdx]: also can we just name the four literals. Saying new is time relative and can go stale. Like saying something will be ready tomorrow. Same link as that other place, https://github.com/goauthentik/authentik/blob/3cb4b9953ef31a267104000df98166594ea0d07f/authentik/events/models.py#L139-L142

File: website/docs/add-secure-apps/access-requests/index.mdx [hunk 30]
```diff
@@ -0,0 +1,160 @@
...
+| -------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Access request       | A request from a specific user for access to an application. A request can be pending, approved, denied, or expired.                                                    |
+| Request rule         | A configuration that determines which request flow is used, who reviews requests, and how reviewers are notified. If no request flow is set, falls back to brand value. |
+| Request rule binding | A binding between a request rule and a requestable application. It also determines which users or groups can request access.                                            |
+| Reviewer             | A user or group authorized to approve or deny an access request.                                                                                                        |
+| Expiry               | The period for which access granted through an approved request remains valid.                                                                                          |
```
Comment (dominic-r) [inline website/docs/add-secure-apps/access-requests/index.mdx]: consider splitting in 2 rows:

Pending expiry; how long an unanswered request survives 
Maximum granted expiry ; how long the grant lasts

File: website/docs/add-secure-apps/access-requests/index.mdx [hunk 31]
```diff
@@ -0,0 +1,160 @@
...
+authentik_enterprise: true
+authentik_version: "2026.8"
+authentik_preview: true
+---
+
+## About access requests
+
+Access requests allow users to request access to an application when they need it, rather than requiring an administrator to provision access in advance.
+
+Each request is evaluated against configured request rules and routed to one or more reviewers. When approved, access is granted for a defined period and automatically revoked when it expires.
```
Comment (dominic-r) [inline website/docs/add-secure-apps/access-requests/index.mdx]:
```suggestion
Each request is evaluated against configured request rules and routed to one or more reviewers. When approved, access is granted for a defined period and expires automatically.
```

Revoked is a distinct status only reached when there's a specific revoke action. Expires keeps the status as approved which makes more sense here

**Review by dominic-r — APPROVED** [review]

File: website/docs/add-secure-apps/access-requests/index.mdx (hunk 25 above)
Comment (dewi-tik) [reply website/docs/add-secure-apps/access-requests/index.mdx:70]: @beryju

File: website/docs/add-secure-apps/access-requests/index.mdx (hunk 24 above)
Comment (dewi-tik) [reply website/docs/add-secure-apps/access-requests/index.mdx]: @BeryJu

File: website/docs/add-secure-apps/access-requests/index.mdx (hunk 25 above)
Comment (dewi-tik) [reply website/docs/add-secure-apps/access-requests/index.mdx:70]: @dewi-tik @dominic-r need to test this - minimum reviewers is expected number of necessary approvals - otherwise it's a bug
